Tests/TuistKitTests/Services/EditServiceTests.swift

```diff
@@ -0,0 +1,86 @@
+import MockableTest
+import TSCBasic
+import TuistCore
+import TuistGraph
+import TuistLoader
+import XcodeProj
+import XCTest
+@testable import TuistCoreTesting
+@testable import TuistKit
+@testable import TuistLoaderTesting
+@testable import TuistPluginTesting
+@testable import TuistSupportTesting
+
+final class EditServiceTests: XCTestCase {
+    var subject: EditService!
+    var opener: MockOpener!
+    var configLoader: MockConfigLoader!
+    var pluginService: MockPluginService!
+    var cacheDirectoriesProvider: MockCacheDirectoriesProvider!
+    var cacheDirectoriesProviderFactory: MockCacheDirectoriesProviderFactory!
+    var projectEditor: MockProjectEditing!
+
+    override func setUpWithError() throws {
+        super.setUp()
+        opener = MockOpener()
+        configLoader = MockConfigLoader()
+        pluginService = MockPluginService()
+        cacheDirectoriesProvider = try MockCacheDirectoriesProvider()
+        cacheDirectoriesProviderFactory = MockCacheDirectoriesProviderFactory(provider: cacheDirectoriesProvider)
```

Comment:
These got deleted in a recent PR: https://github.com/tuist/tuist/pull/6326

You will need to update to the latest `main`, otherwise, merging this would make `main` fail as we don't have a merge queue right now.